core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -288,6 +288,10 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     StartsWithTech("Starts with [tech]", UniqueTarget.Nation),
     ResearchableMultipleTimes("Can be continually researched", UniqueTarget.Global),
 
+    BaseUnitSupply("[amount] Unit Supply", UniqueTarget.Global),
+    UnitSupplyPerPop("[amount] Unit Supply per [amount] population", UniqueTarget.Global), 
```

Comment:
This feels like it should have a cityfilter

Reply:
I was planning so at first too, but removed it after deciding that would complicate the calculation for supply a lot. Basically for each unique I would have to iterate over all cities and add their population only if the city matched the filter (O(n^2)). If you want I can still add it in

Reply:
I think we should add it, it's O(n) per unique which is exactly the complexity of a unique that sums population count anyway :)

Reply:
Should be added now